content/concepts/introduction/software-ecosystem.md

```diff
@@ -0,0 +1,61 @@
+# Software Ecosystem
+
+The XRP Ledger is home to a deep, layered ecosystem of software projects powering and enabling an Internet of Value. It's impossible to list every project, tool, and business that interacts with the XRP Ledger, so this page only lists a few categories and highlights some central projects that are documented here on [xrpl.org](https://xrpl.org).
+
+## Stack Levels
+
+[![Ecosystem diagram with the four layers: XRP Ledger peer-to-peer network on the bottom, Programming Libraries above that, Middleware next, and Apps and Services at the top](img/ecosystem.png)](img/ecosystem.png)
+
+- The [basis of the XRP Ledger](#rippled-the-core-server) is a peer-to-peer network of always-on servers sharing transactions, engaging in the [consensus process](consensus.html) and processing [transactions](transaction-basics.html). Everything else in the XRP Ledger ecosystem is ultimately built on top of this peer-to-peer network, directly or indirectly.
+
+- [_Programming Libraries_](#programming-libraries) exist in higher level software, where they are imported directly into program code, and contain premade implementations of routines to access the XRP Ledger.
+
+- [_Middleware_](#middleware) provides indirect access to XRP Ledger data. Applications in this layer frequently have their own data storage and processing.
+
+- [_Apps and Services_](#apps-and-services) provide user-level interaction with the
+
+
+### rippled: The Core Server
+
+The peer-to-peer network at the heart of the XRP Ledger requires a highly-reliable, efficient server to enforce the rules of consensus and transaction processing. The reference implementation of this server software, [**`rippled`**](the-rippled-server.html) (pronounced "ripple-dee") is managed and published by Ripple, and available under a permissive open-source license so anyone can inspect and modify their own instance of the server.
```

Comment:
I object at “managed and published”. Ripple publishes its own version, but we should not claim we manage `rippled` which is an open-source project.

Reply:
I don't think it's inaccurate to say Ripple manages _the reference implementation_, unless there are non-Ripple employees who have commit access to the `rippled` repo.

Reply:
To make it clearer (and improve readability too!) I changed this to the following:

> Ripple manages and publishes a reference implementation of this server software, called [**`rippled`**](the-rippled-server.html) (pronounced "ripple-dee"). The server is available under [a permissive open-source license](https://github.com/ripple/rippled/blob/develop/LICENSE), so anyone can inspect and modify their own instance of the server, and re-publish with few restrictions.